src/test/java/io/harness/cf/client/api/EvaluatorIntegrationTest.java

```diff
@@ -96,7 +97,8 @@ public void prepareTestData() {
 
       for (final String key : file.getExpected().keySet()) {
         final Object expected = file.getExpected().get(key);
-        final TestCase testCase = new TestCase(file.getTestFile(), key, expected, file);
+        final String testName =  removeExtension(file.getTestFile());
```

Comment:
every file have more test cases based on target input, so test case name should be name + target

Reply:
Yep - I've handled it down below see: https://github.com/harness/ff-java-server-sdk/pull/80/files#diff-713ec79420dff87b6848e80bccadcea3dea132f10aedf548f21c14b459590781R29